### ISSUE TITLE
Problem when packer started

### ISSUE THREAD

**Issue by tariky** [hayes0724/shopify-packer#37]
When I start packer, I got this error: //<\/script>".replace("HOST", location.hostname)); //]]>.

**Comment by batjeciftci** [comment]
@tariky this error occurs because of the script tag isn't closed correctly. Just find the tag (I think it was inside theme.liquid at the bottom) and close it correctly.

**Comment by tariky** [comment]
> @tariky this error occurs because of the script tag isn't closed correctly. Just find the tag (I think it was inside theme.liquid at the bottom) and close it correctly.

I tried that, there is none script tags in theme.liquid.

**Comment by hayes0724** [comment]
@tariky Check the version (and post in issue) of packer and packer-blank-theme that you are using. Also the steps needed to recreate, examples:
- started a new project with packer init
- worked on an existing project/website
- any commands or files changed

**Comment by mrpacman101** [comment]
> > @tariky this error occurs because of the script tag isn't closed correctly. Just find the tag (I think it was inside theme.liquid at the bottom) and close it correctly.
> 
> I tried that, there is none script tags in theme.liquid.

If you don't find the script tag in your theme.liquid maybe you have not added it yet. Make sure you added this to the bottom of your theme.liquid file:
```
 {%- render 'script-tags' -%}
 {%- render 'style-tags' -%}
```

I use render instead because include was flagged as deprecated by shopify: 
https://shopify.dev/docs/themes/liquid/reference/tags/deprecated-tags#include

Reference in Documenation:
https://github.com/hayes0724/shopify-packer#script-and-style-tags

**Comment by hayes0724** [comment]
Source of the issue was the body tag, class was not closed:
```
<body class="template-{{ template.name }}{%- if template.suffix -%}__{{ template.suffix }}{%- endif -%}>
```
missing " at the end.

Thanks for the feedback, this is fixed in packer theme [v1.0.9](https://github.com/hayes0724/packer-blank-theme/releases/tag/v1.0.9)
